src/commands/ui.rs

```diff
@@ -38,12 +38,16 @@ pub fn show_ui(options: &Options) -> anyhow::Result<()> {
             }
         }
     }
-    if open::that(&url).is_ok() {
-        Ok(())
-    } else {
+    if !args.url && !open::that(&url).is_ok() {
```

Comment:
Why we don't print error returned by `open::that` ? Sometimes users may want to see why we couldn't open a browser.

Reply:
Fixed!